### PR TITLE
kubecolor: Add version 0.2.1

### LINKED ISSUE #5424: [Request]: kubecolor
### Prerequisites

- [X] I have searched all issues/PRs to ensure it has not already been reported or fixed.

### Criteria

- [X] Non-GUI tool
- [X] Reasonably well-known and widely used (e.g. if it's a GitHub project, it should have at least 500 stars and/or 150 forks)
- [X] English interface (or at least English documentation)
- [X] Latest stable version
- [X] Full version (i.e. not a trial version)
- [X] Fairly standard install (e.g. uses a version-specific download URL, no elaborate pre/post install scripts)

### Name

kubecolor

### Description

Wraps kubectl with colorful output

### Homepage

https://github.com/kubecolor/kubecolor

### Download Link(s)

https://github.com/kubecolor/kubecolor/releases

### Some Indication of Popularity/Repute

Original repo, [hidetatz/kubecolor](https://github.com/hidetatz/kubecolor), has 1.4k stars and 105 forks. That repo is now archived.

Successor repo, [kubecolor/kubecolor](https://github.com/kubecolor/kubecolor), only has 135 stars and 8 forks, but this is because the repo was recreated and not transferred.

This package was previously requested to the Main bucket in #5202, with PR #5378. But those [were closed](https://github.com/ScoopInstaller/Main/issues/5202#issuecomment-1869135288) in favor of the issue (https://github.com/ScoopInstaller/Extras/issues/12492) and PR (https://github.com/ScoopInstaller/Extras/pull/12493) to add the package to the Extras bucket.

Then @HUMORCE requested that they be added to the Main bucket (https://github.com/ScoopInstaller/Extras/issues/12492#issuecomment-1890594596), so here we are back at this repo again :)

### PR DESCRIPTION
**Pull request by applejag**
Closes #5424

- [x] I have read the [Contributing Guide](https://github.com/ScoopInstaller/.github/blob/main/.github/CONTRIBUTING.md).

### CODE REVIEW

**Comment by prune998** [comment]
Kubecolor v0.2.1 is out now with some bug correction. Better update this PR with the new release before merging. Thx again for your work @applejag !

File: bucket/kubecolor.json [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+        },
+        "arm64": {
+            "url": "https://github.com/kubecolor/kubecolor/releases/download/v0.2.1/kubecolor_0.2.1_windows_arm64.zip",
+            "hash": "620ecf1d2bf2705cb14494667785265c66dd2f5afe8dbbfa841507cea1cf30c9"
+        }
+    },
+    "bin": "kubecolor.exe",
+    "checkver": {
+        "github": "https://github.com/kubecolor/kubecolor"
+    },
```
Comment (HUMORCE) [inline bucket/kubecolor.json:22]:
```suggestion
    "checkver": "github",
```

Better, but it doesn't matter.

**Review by HUMORCE — APPROVED** [review]

**Pull request merged**